astroplan/core.py

```diff
@@ -1342,14 +1343,14 @@ def __init__(self, low, high):
         raise NotImplementedError
 
 
-class AboveAirmass(Constraint):
+class AirmassRange(Constraint):
     """
-    An object containing an airmass lower limit.
+    Constrain the airmass within a given range.
     """
 
-    def __init__(self, low):
+    def __init__(self, upper_limit, lower_limit=1):
```

Comment:
And in case it's not clear, I'm perfectly happy with allowing ``AirmassRange(2, 1.1)``: even though I don't know of any use case there might be an edge one.  I just don't think we should make the much more common ``AirmassRange(2)`` suffer because of the option for that.